ksqldb-engine/src/main/java/io/confluent/ksql/engine/generic/GenericRecordFactory.java

```diff
@@ -146,7 +146,8 @@ private static LogicalSchema withRowTime(final LogicalSchema schema) {
           column,
           functionRegistry,
           config,
-          "insert value").resolve(valueExp);
+          "insert value",
+          false).resolve(valueExp);
```

Comment:
i believe this is used on the pull query path as well as the insert into path, in which case should we be using the interpreter? might be worth confirming the code flow

Reply:
thanks! I must've confused the two